.github/pull_request_template.md

```diff
@@ -0,0 +1,72 @@
+# Description
+<!---  Write a brief description of what your code does and how it relates to the issue it is resolving or feature enhancement it is implementing. -->
+
+# Details
+#### URL to JIRA issue
+N/A
+<!---
+  Delete this comment and include the URL of the JIRA issue the pull request is related to.
+  If no JIRA issue exists for this PR, replace this comment with N/A.
+  Your pull request will not pass the required checks if this is not followed.
+-->
+
+#### Link to staging deployment URL 
+<!---
+  Delete this comment and include the URL of the staging environment for this pull request.
+  If no staging environment exists for this PR, replace this comment with N/A and explain why
+  a staging environment is not necessary.
+  Your pull request will not pass the required checks if this is not followed.
+-->
+
+#### Links to any PRs OR internal resources related to this PR
+<!---
+  Delete this comment and include the URLs of any pull requests that are related to this PR.
+  Place each PR on a new line. If no other PRs relate to this PR, replace this comment with N/A.
+  Your pull request will not pass the required checks if this is not followed.
```

Comment:
why will it not pass?

Reply:
It will, I just copy-pasted it. Now changed.